device/samples/helpers/device-twin-service/twin-service.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const iothub = require('azure-iothub');
+const connectionString = process.env.IOTHUB_CONNECTION_STRING;
+const deviceId = 'devDevice';
+
+if (connectionString === undefined) {
+  console.log('IoT Hub connection string not set');
+  process.exit(-1);
```

Comment:
Can you explain what a negative exit code does? I haven't seen this before.

Reply:
I don't think it does anything special :)

i changed it to zero

Reply:
A 0 exit code means success so I think it should be non-zero